demos/demos_index.rst

```diff
@@ -0,0 +1,17 @@
+
+.. _demos-reference-samples:
+
+OpenAMP Samples and Demos
```

Comment:
Should this table of contents also include a link to the  OpenAMP Inter Process Demos (inter_process.rst)?